dist/eas.d.ts

```diff
@@ -3,7 +3,8 @@ import { Delegated, Offchain } from './offchain';
 import { AttestationRequest, DelegatedAttestationRequest, DelegatedProxyAttestationRequest, DelegatedProxyRevocationRequest, DelegatedRevocationRequest, MultiAttestationRequest, MultiDelegatedAttestationRequest, MultiDelegatedProxyAttestationRequest, MultiDelegatedProxyRevocationRequest, MultiDelegatedRevocationRequest, MultiRevocationRequest, RevocationRequest } from './request';
 import { Base, SignerOrProvider, Transaction } from './transaction';
 import { EAS as EASContract } from '@ethereum-attestation-service/eas-contracts';
-import { BigNumber, BigNumberish } from 'ethers';
+import { BigNumber, BigNumberish, Overrides, PayableOverrides } from 'ethers';
```

Comment:
I wonder why generated files under `dist` are committed to git.

Reply:
Hey @raymondfeng,
This is a common practice when you want to support users to be able to use the package directly from GitHub (e.g., early adapters), without requiring them to be able to build it themselves (e.g., non-TypeScript projects).